src/streamwatch/forms.py

```diff
@@ -71,7 +71,7 @@ class SetupForm(forms.Form):
     collect_tz = forms.ChoiceField(
         required=False,
         label='Timezone',
-        choices=[(None,'')] + timeutils.make_tz_tuple_list(),   
+        choices=[(None,'')] + timeutils.make_tz_tuple_list(timeutils.tz_key_shortlist, datetime(2022,1,1)),   
```

Comment:
`datetime` call needs to be updated to a `datetime.datetime` call. 

We imported the `datetime` module so you need to reference the datetime class in the datetime module